include/nlohmann/json.hpp

```diff
@@ -3695,15 +3695,25 @@ class basic_json // NOLINT(cppcoreguidelines-special-member-functions,hicpp-spec
             if (idx >= m_value.array->size())
             {
 #if JSON_DIAGNOSTICS
-                // remember array size before resizing
+                // remember array size & capacity before resizing
                 const auto previous_size = m_value.array->size();
+                const auto old_capacity = m_value.array->capacity();
```

Comment:
Please call this `previous_capacity` to match the style of `previous_size` variable name.

Reply:
Everywhere else it's called `old_capacity`, do you want me to rename all instances of `old_capacity` to `previous_capacity` (for consistency)?

Reply:
Yes, you are right. Sorry - I only looked at these few lines. Then I would ask you to rename `previous_size` to `old_size` for consistency with the larger part of the code.

Reply:
No worries! Renamed and amended my commit.